docker/README.md

```diff
@@ -0,0 +1,38 @@
+# Containerizing Dell EMC OpenManage Ansible Modules
+Containerized image of Dell EMC OpenManage Ansible Modules can be built using the dockerfile available [here](./Dockerfile). We will be using Docker Engine for containerizing OpenManage Ansible Modules. More information about Docker Engine and install instructions can be found [here](https://docs.docker.com/engine/).
+
+## Build
+Download [Dockerfile](./Dockerfile) to the machine where Docker Engine is installed and run below command. 
+```bash
+docker build -t dellemc/openmanage-ansible-modules .
+```
+Upon success, new image with name _dellemc/openmanage-ansible-modules_ will be created. One can verify successful creation of image using command `docker image ls` which will list newly created image with repository name as _dellemc/openmanage-ansible-modules_.
+
+Note: 
+1. This command need to be run from same directory where Dockerfile is downloaded. If not use the option `--file` to specify the path to Dockerfile.
+1. Docker file shown here uses the Docker official image [python:3](https://hub.docker.com/_/python) as base. Change this value is you want to use other python images as base. 
+
+## Usage
+`dellemc/openmanage-ansible-modules` docker image contains no playbook or configuration. One need to pass playbook, var files and inventory for running a task
+
+### Ansible inventory, vars_files and playbooks
+Setup a volume or use a bind mount folder containing ansible inventory, vars_files and playbook. This need to be mounted into the container using -v or --mount.
+
+Modules are configured to run with python in `/usr/local/bin/python`. Hence 'ansible_python_interpreter' option need to be set in inventory file or passed as argument while running the playbook with `-e` option
+
+### Running playbook
+Once playbook, inventory and variable files are set in the current working directory, one can run the playbook as shown below
+```bash
+docker run --rm \
```

Comment:
Do we need to access Docker Host Resources/artifact from docker host frequently? How do we planning to manage share access required for modules such export system configuration profiles . Will this share created already inside container?

Reply:
will update readme with mount instructions

Reply:
@saishivarcr, yup.. lets be specific on what details we need/using from `Docker Host` since host is something shared by many dockers. Also the specific access rights we need on host. It should end up like customer should run generated images in privileged mode

Reply:
@rajeevarakkal privileged mode is not required as share will not be mounted inside the container. Share has to be mounted on docker engine and bind mounted in the container at `/dellemc`. These instructions are already updated. Also default access rights are sufficient in this case, hence not documented.